Home.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="Styles.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio | Home</title>
+</head>
+
+<body>
+    <header>
+        <a href="Home.html" class="Homebutton">
+            James
+        </a>
+        <nav class="topnav">
+            <a href="#">
+                About
+            </a>
+            <a href="#">
+                Portfolio
+            </a>
+            <a href="#">
+                Contact Me
+            </a>
+        </nav>
+        <nav class="sociallinks">
+            <a href="https://github.com/James-Benson">
+                <i class="fab fa-github"></i>
+            </a>
```

Comment:
Do these each need to be on a new line? If you had the opening tag, content and closing tag on one line the end result would still be the same and your code would take up less space.